syncapi/storage/sqlite3/syncserver.go

```diff
@@ -106,3 +119,172 @@ func (d *SyncServerDatasource) prepare() (err error) {
 	}
 	return nil
 }
+
+func (d *SyncServerDatasource) Events(ctx context.Context, eventIDs []string) ([]gomatrixserverlib.HeaderedEvent, error) {
```

Comment:
I'm confused by all of these functions here that are already in `shared`. Is this to do with the GOID tracing? If so, can we revert those commits?

Reply:
Yes it's for goid tracing with dbctx. Should be removed.

Reply:
I've backed this out now.